docs/core/compatibility/core-libraries/8.0/getfolderpath-unix.md

```diff
@@ -0,0 +1,62 @@
+|---------------------|---------------------------|--------------------------------------------------------------------|
+| `MyDocuments`       | `$HOME`                   | Uses `XDG_DOCUMENTS_DIR` if available; otherwise `$HOME/Documents` |
+| `Personal`          | `$HOME`                   | Uses `XDG_DOCUMENTS_DIR` if available; otherwise `$HOME/Documents` |
+
+### macOS
+
+| SpecialFolder value | Path (.NET 7 and earlier) | Path (.NET 8 and later) |
+|-|-|-|
+| `MyDocuments` | `$HOME` | [NSDocumentDirectory](https://developer.apple.com/documentation/foundation/nssearchpathdirectory/nsdocumentdirectory) (`$HOME/Documents`) |
+| `Personal` | `$HOME` | [NSDocumentDirectory](https://developer.apple.com/documentation/foundation/nssearchpathdirectory/nsdocumentdirectory) (`$HOME/Documents`) |
+| `ApplicationData` | `$HOME/.config` | [NSApplicationSupportDirectory](https://developer.apple.com/documentation/foundation/nssearchpathdirectory/nsapplicationsupportdirectory) (Library/Application Support) |
+| `LocalApplicationData` | `$HOME/.local/share` | [NSApplicationSupportDirectory](https://developer.apple.com/documentation/foundation/nssearchpathdirectory/nsapplicationsupportdirectory) (Library/Application Support) |
+| `MyVideos` | `$HOME/Videos` | [NSMoviesDirectory](https://developer.apple.com/documentation/foundation/nssearchpathdirectory/nsmoviesdirectory) (`$HOME/Movies`) |
+
+### Android
+
+| SpecialFolder value | Path (.NET 7 and earlier) | Path (.NET 8 and later) |
+|---------------------|---------------------------|-------------------------|
+| `MyDocuments`       | `$HOME`                   | `$HOME/Documents`       |
+| `Personal`          | `$HOME`                   | `$HOME/Documents`       |
+
+## Version introduced
+
+.NET 8 Preview 1
+
+## Type of breaking change
+
+This change is a [behavioral change](../../categories.md#behavioral-change).
+
+## Reason for change
+
+The previous behavior was incorrect and didn't meet user expectations for Linux, macOS, and Android.
+
+## Recommended action
+
+The most common break is if you're passing <xref:System.Environment.SpecialFolder.Personal?displayProperty=fullName> to <xref:System.Environment.GetFolderPath(System.Environment.SpecialFolder)?displayProperty=nameWithType> on Unix to get the `$HOME` directory (`Environment.GetFolderPath(Environment.SpecialFolder.Personal)`). <xref:System.Environment.SpecialFolder.Personal?displayProperty=nameWithType> and <xref:System.Environment.SpecialFolder.MyDocuments?displayProperty=nameWithType> are aliases for the same underlying enumeration value. If you're using <xref:System.Environment.SpecialFolder.Personal?displayProperty=nameWithType> in this way, change your code to pass <xref:System.Environment.SpecialFolder.UserProfile?displayProperty=nameWithType> instead (`Environment.GetFolderPath(Environment.SpecialFolder.UserProfile)`).
```

Comment:
The issue listed `SpecialFolder.Documents`, but that value doesn't seem to exist, so I used `SpecialFolder.MyDocuments`. Is that correct?

Reply:
Yes, that was a mistake/type-o in the original issue's Recommended Action. `Environment.SpecialFolder.Documents` => `Environment.SpecialFolder.MyDocuments`.